easybuild/easyconfigs/a/annovar/annovar-2016Feb01-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'annovar'
+version = '2016Feb01'
```

Comment:
mention how you came to this (`$DATE` in `table_annovar.pl`?)

also, I'm more in favour of `20160201`, since that'll sort properly